src/math/abs.jl

```diff
@@ -0,0 +1,59 @@
+
+"""
+    abs_val(num)
+
+Program to find the absolute value of a number
+
+# Example
+```julia
+abs_val(-100) # returns 100
+abs_val(0) # returns 0
+abs(123.1) # returns 123.1
+-1000 == abs_val(-1000) #returns false
+1000 == abs_val(1000) #returns true
+```
+
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+"""
+function abs_val(num::Int64)
```

Comment:
One of the good point of Julia is that you can be very general with types in function and methods without losing performance. So, you can opt for no type here, or `Int` for the biggest integer type available on the architecture you are testing.
It will make the test in the CI work flawlessly too, which is nice!